pkg/summary/collector.go

```diff
@@ -40,7 +40,15 @@ type LogCollector interface {
 
 type logFunc func(msg string, fields ...zap.Field)
 
-var collector = newLogCollector(log.Info)
+var collector = func() LogCollector {
+	conf := new(log.Config)
+	// Always output summary to stdout.
+	logger, _, err := log.InitLogger(conf)
+	if err != nil {
+		logger = log.L()
+	}
+	return newLogCollector(logger.Info)
```

Comment:
I think we also need to output it into log file if user specify log file

Reply:
It's difficult to know whether the global logger will output to stdout or output to a file here.